src/metadata/bpm.cpp

```diff
@@ -0,0 +1,60 @@
+#include "metadata/bpm.h"
```

Comment:
We already have a folder for track related objects -- "track". Could you move this in there instead of 'metadata'?